### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by zoracon**
Update Nonfunctional Hosts Standard

We have this in generated rules with make-trivial-rule script. It should be reflected in the Docs

Related: #14936

### CODE REVIEW

**Comment by pipboy96** [comment]
@zoracon Let's change `make-trivial-rule` to match the docs instead, since we usually use that format.

**Comment by pipboy96** [comment]
Also, is there evidence that this script is still used by people? Maybe we can completely remove it?

**Comment by zoracon** [comment]
> Also, is there evidence that this script is still used by people? Maybe we can completely remove it?

@pipboy96 I am not sure who uses this and who doesn't. I am going to research the background on why this comment for hosts was structured this way. There's a lot of discussion from a few years ago concerning this topic. I want to decide on a simplistic format going forward.

So my question is the format in `make-trivial-rule` more useful or is the more verbose format in the document better to keep?

**Comment by zoracon** [comment]
To add, we also reference the `make-trivial-rule` tool in our Update Channels documentation
https://github.com/Hainish/https-everywhere/blob/master/docs/en_US/ruleset-update-channels.md

Which being actively experimented with at the moment.

**Comment by cschanaj** [comment]
The style in make-trivial-rule was being discussed in #6868 and the _agreement_ (tagged master list) was made by a number of contributors. However, I agree with @pipboy96  that the style in the documentation (grouped list) and its variants are more commonly seen. Personally, I prefer the style in the current documentation. 

I believe the grouped list styles gained its popularity after the agreement is made. There was a time many contributors created their own bots for automatic generation of rulesets (#7691), and the grouped list styles turned out to be easier to parsers and formatters (e.g. https://github.com/EFForg/https-everywhere/issues/6912#issuecomment-248211667) 

cc @jeremyn @J0WI @Hainish

**Comment by zoracon** [comment]
Thank you for the background @cschanaj. I am inclined to go with the previous consensus instead of going down that path again. The conversation seemed to have come to a good middle ground between the styles. I know the more verbose categorization seems to be dominant, but going forward would like to create a script that can easily parse this out and enforce this style.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -281,25 +281,20 @@ It is useful to list hosts that do not work in the comments of a `ruleset`.  Thi
...
+		forum2.glavbukh.ru (incomplete certificate chain)
+	Redirect to HTTP:
+		8marta2013.glavbukh.ru
+		den.glavbukh.ru
+	Refused:
+		e.glavbukh.ru
+		www.e.glavbukh.ru
+	Time out:
+		psd.glavbukh.ru
+		str.glavbukh.ru
```
Comment (J0WI) [inline CONTRIBUTING.md:297]: Should we use an [example domain](https://www.iana.org/domains/reserved) or a domain we control for this?

Comment (zoracon) [reply CONTRIBUTING.md:297]: We can, I can adjust to reflect that or you can.

Comment (J0WI) [reply CONTRIBUTING.md:297]: #19899 feel free to adjust/copy my changes.

**Comment by zoracon** [comment]
Merged in related PR, closing this one out

**Pull request closed without merging**